azext_iot/tests/iothub/test_iot_ext_int.py

```diff
@@ -22,7 +23,7 @@
     req_env_set=ENV_SET_TEST_IOTHUB_BASIC, opt_env_set=opt_env_set
 )
 
-LIVE_HUB = settings.env.azext_iot_testhub
+LIVE_HUB = settings.env.azext_iot_testhub if settings.env.azext_iot_testhub else "test-hub-" + str(uuid4())
```

Comment:
We have a number of tests (most if not all in this module) that require the hub to be configured with a storage account. For new hubs that are created per test, the configuration with storage needs to be done as part of test setup.

Reply:
My thought was that why not use a fixed shared storage account, instead of creating a separate one. Do you see any downsides of that

Reply:
I'm OK with that but the fixed storage account still needs to be configured to work with a new IoT Hub.